HoloPipelines/pipelines/components/compDicom2numpy.py

```diff
@@ -88,7 +88,7 @@ def reorientateNumpyList(numpyList):
     # transpose numpy i.e. (z, y, x) ---> (x, y, z)
     numpyList = numpyList.transpose(2, 1, 0)
     numpyList = np.flip(numpyList, 0)
-    numpyList = np.flip(numpyList, 1)
+    # numpyList = np.flip(numpyList, 1) # need to confirm this
```

Comment:
Can you confirm this today as you will be unavailable for the next 6 days? Otherwise it's hard to accept this PR

Reply:
Confirmed, NumPy array is now in correct orientation. Commented code removed

Reply:
thanks